docs/lookalikes/3configuration-testing.md

```diff
@@ -0,0 +1,26 @@
+---
+id: configuration-testing
+slug: /lookalikes/configuration-testing
+---
+
+# Configuration Tester
+
+The configuration tester is a simple way to test against all installed (and enabled) lookalikes on this channel or preset.
+
+![lookalike tester example](../../static/img/lookalikes/configuration-tester-step1.png)
+
+## Ensure Lookalikes are Installed
+
+The currently installed and enabled lookalikes on my channel for this test is `wigglez_morse`, `wigglez_braille`, and `wigglez_global`.
```

Comment:
The currently installed and enabled lookalikes on my channel for this test `are` wigglez_morse, wigglez_braille, and wigglez_global.